setup.py

```diff
@@ -61,7 +61,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "mlflow",
         ],
         # TODO(hvy): Unpin `sphinx` version after https://github.com/sphinx-doc/sphinx/issues/7807.
-        "document": ["sphinx>=3.0.0,!=3.1.0,!=3.1.1", "sphinx_rtd_theme"],
+        "document": ["sphinx>=3.0.0,!=3.1.0,!=3.1.1", "sphinx_rtd_theme<0.5.0"],
```

Comment:
`<0.5.0` is required to prevent installing the latest version `sphinx_rtd_theme` on CircleCI.

Reply:
How about adding a comment here to show that the version of `sphinx_rtd_theme` is restricted due to the ReadTheDocs?